assembl/models/configurable_fields.py

```diff
@@ -86,6 +86,8 @@ class AbstractConfigurableField(DiscussionBoundBase):
 
     required = Column(Boolean(), default=False)
 
+    hidden = Column(Boolean(), default=False)
```

Comment:
You should put also `server_default='0'` here so it's the same definition as the alembic step. You can keep the default=False here too.